src/odemis/driver/tescan.py

```diff
@@ -1086,38 +1099,68 @@ def _updatePosition(self):
         """
         update the position VA
         """
-        x, y, z, rot, tilt = self.parent._device.StgGetPosition()
+        x, y, z, rz, rx = self.parent._device.StgGetPosition()
         self._position["x"] = -x * 1e-3
         self._position["y"] = -y * 1e-3
         self._position["z"] = -z * 1e-3
 
+        if set(self.axes.keys()).intersection({"rz", "rx"}):
+            self._position["rz"] = self._degrees_to_rad(rz)
+            self._position["rx"] = self._degrees_to_rad(rx)
+
         # it's read-only, so we change it via _value
         pos = self._applyInversion(self._position)
         self.position._set_value(pos, force_write=True)
         logging.debug("Updated stage position to %s", pos)
 
-    def _doMoveAbs(self, pos):
+    def _degrees_to_rad(self, val: float) -> float:
         """
-        move to the position
+        change the value of val from degrees to radians
+        :param val: float value in degrees to convert
+        :return: float value in radians
+        """
+        return math.radians(val)
+
+    def _rad_to_degrees(self, val: float) -> float:
+        """
+        change the value of val from radians to degrees
+        :param val: float value in radians to convert
+        :return: float value in degrees
+        """
+        return math.degrees(val)
```

Comment:
Yeah I agree on that. At first I though it would make the code much better readable and in addition if these methods ever need any more code they would already be there. But for now indeed lets stick to directly using the math functions.